kalitectl.py

```diff
@@ -699,7 +701,10 @@ def docopt(doc, argv=None, help=True, version=None, options_first=False):  # @Re
     # Since positional arguments should always come first, we can safely
     # replace " " with "=" to make options "--xy z" same as "--xy=z".
     arguments = docopt(__doc__, version=str(VERSION), options_first=False)
-    
+
+    settings_module = arguments.pop('--settings', None)
+    os.environ['DJANGO_SETTINGS_MODULE'] = settings_module
```

Comment:
This probably requires that `--settings=X` be specified prior to any actual DJANGO_OPTIONS, due to the ordering constraint currently in place for `kalite` command vs Django options, right? Probably not a big deal, and I imagine it will tell the user if there's an ordering issue, but might be worth testing a few different cases of ordering (especially any we have in the docs) to make sure it handles things nicely.